docs/content/tutorial/step_03.ngdoc

```diff
@@ -88,6 +88,19 @@ Let's see an example:
     });
 ```
 
+```html
+<html ng-app="myApp">
+<head>
+  <script src="bower_components/angular/angular.js"></script>
+  <script src="app.js"></script>
+</head>
+<body>
+   <!-- The following line is how to use the greetUser angular.js component above in your html doc-->
+  <greet-user></greet-user>
+ </body>
+</html>
+```
```

Comment:
Including a whole HTML page just to show one line is distracting imo. I would shorten it to something like:

```html
<body>
  <!-- ... -->
  <greet-user></greet-user>
</body>
```